### DISCUSSION TITLE
如何才能开发一个启动引导界面

### DISCUSSION THREAD

**Discussion by hanbingzi** [opensumi/core#3097] [Q&A] [answered]
如下图，我希望也能开发一个类似的启动引导界面，大致思路是如何做那？
<img width="1728" alt="image" src="https://github.com/opensumi/core/assets/20720644/43802d8e-a2da-4fdc-a7bb-f4fb82248523">

<img width="1000" alt="image" src="https://github.com/opensumi/core/assets/20720644/5ea0b8d0-9d31-4d92-ad6d-78aa32b5a0b3">

**Comment by erha19** [comment, marked as answer]
@hanbingzi 可以参考一下 https://github.com/opensumi/app-desktop 这个项目

**Reply by hanbingzi** [reply]
ok，谢谢
